Backend/router/v1/report.ctrl.js

```diff
@@ -0,0 +1,67 @@
+const models = require('../../models');
+
+const create = (req, res) => {
+  models.report.create({
+    what: req.body.data[0].what,
+    location: req.body.data[0].location,
+    time: req.body.data[0].time,
+    who: req.body.data[0].who,
+    details: req.body.data[0].details,
+  }).then(result => {
+    console.log(result);
+    console.log("success!");
+  });
+  res.status(201);
+  res.send("good");
+}
+
+const list = (req, res) => {
+  models.report.findAll().then((result) => {
+    res.json(result);
+  }).catch((err) => {
+    console.log("error!!!!!!!!!!");
+  });
+}
+
+
+const read = (req, res) => {
+  const report_ID = parseInt(req.params.report_ID, 10) //decimal number
```

Comment:
How about we follow more common naming convention? (came case) https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

Reply:
@shberry  I modified it. Please cheak.